android/app/build.gradle

```diff
@@ -111,7 +111,7 @@ def enableProguardInReleaseBuilds = false
  * give correct results when using with locales other than en-US.  Note that
  * this variant is about 6MiB larger per architecture than default.
  */
-def jscFlavor = 'org.webkit:android-jsc:+'
+def jscFlavor = 'org.webkit:android-jsc-intl:+'
```

Comment:
Can you please explain what this is / does?

Reply:
It is needed to use **Intl.collator** properly so that we could sort based on character-specific letters (as I understood it makes it more internalized) Here is a link:  https://stackoverflow.com/questions/56943813/using-intl-properly-in-android-react-native-app